app/Models/Author.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Author extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'name', 'email', 'github', 'twitter'
+    ];
+
+    public static $rules = [
```

Comment:
delete rules from here.

Reply:
added rules in separate file

Reply:
delete rules from this model. i can still see it here